Source/WebCore/platform/graphics/GlyphPage.h

```diff
@@ -92,20 +95,28 @@ class GlyphPage : public RefCounted<GlyphPage> {
     GlyphData glyphDataForIndex(unsigned index) const
     {
         Glyph glyph = glyphForIndex(index);
-        return GlyphData(glyph, glyph ? &m_font : nullptr);
+        bool isColor = isColorForIndex(index);
+        return GlyphData(glyph, glyph ? &m_font : nullptr, isColor);
     }
 
     Glyph glyphForIndex(unsigned index) const
     {
-        ASSERT_WITH_SECURITY_IMPLICATION(index < size);
+        RELEASE_ASSERT_WITH_SECURITY_IMPLICATION(index < size);
         return m_glyphs[index];
     }
 
+    bool isColorForIndex(unsigned index) const
+    {
+        RELEASE_ASSERT_WITH_SECURITY_IMPLICATION(index < size);
+        return m_isColor.get(index);
+    }
+
     // FIXME: Pages are immutable after initialization. This should be private.
-    void setGlyphForIndex(unsigned index, Glyph glyph)
+    void setGlyphForIndex(unsigned index, Glyph glyph, bool isColor)
```

Comment:
Not a big fan of the `bool` argument, when there are call sites that are passing in `false` and it's not clear from reading what it means.